services/121-service/src/registration/validators/any-valid-body.validator.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  createParamDecorator,
+  ExecutionContext,
+  ValidationPipe,
+} from '@nestjs/common';
+
+const RawBody = createParamDecorator(
+  (_data: unknown, ctx: ExecutionContext): any => {
+    const request = ctx.switchToHttp().getRequest();
+    return request.body;
+  },
+);
+
+// This custom decorator is needed to allow external callbacks with unknown or extra properties
+// It applies a ValidationPipe with relaxed settings, so the global whitelist does not block valid payloads
+// Use this for endpoints where you do not control the exact payload structure (e.g., external integrations)
+// The reason that this is needed is that the global ValidationPipe cannot be overridden for specific endpoints.
+export const AnyValidBody = (...pipes: unknown[]) =>
+  RawBody(
+    new ValidationPipe({
+      validateCustomDecorators: true,
+      whitelist: true,
+      skipMissingProperties: true,
+      forbidUnknownValues: false,
+      transform: true,
+    }),
+    ...pipes,
```

Comment:
Did you test this functionally? And does it make sense to add a test that asserts scenarios for both correct / uncorrect payloads with both global/relaxed endpoint?